### PR TITLE
Add new change

### PR DESCRIPTION
**Pull request by dangthanhphuc**

### CODE REVIEW

**Review by dangthanhphuc — COMMENTED** [review]
Finish

File: angular.json [hunk 1]
```diff
@@ -1,107 +1,106 @@
...
-                  "maximumError": "4kB"
+    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+    "version": 1,
+    "newProjectRoot": "projects",
+    "projects": {
+        "rent-ride-fe": {
+            "projectType": "application",
+            "schematics": {
+                "@schematics/angular:component": {
+                    "style": "scss"
```
Comment (dangthanhphuc) [inline angular.json:10]: Comment

**Review by dangthanhphuc — COMMENTED** [review]
Review

File: angular.json (hunk 1 above)
Comment (dangthanhphuc) [reply angular.json:10]: dfsdf

**Comment by dangthanhphuc** [comment]
mnk

File: angular.json [hunk 2]
```diff
@@ -1,107 +1,106 @@
...
-                {
-                  "type": "anyComponentStyle",
-                  "maximumWarning": "2kB",
-                  "maximumError": "4kB"
+    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+    "version": 1,
+    "newProjectRoot": "projects",
+    "projects": {
+        "rent-ride-fe": {
+            "projectType": "application",
```
Comment (phucdang01235) [inline angular.json:7]: Ok

**Review by phucdang01235 — APPROVED** [review]

File: angular.json (hunk 2 above)
Comment (dangthanhphuc) [reply angular.json:7]: Thius be ok

File: angular.json [hunk 3]
```diff
@@ -1,107 +1,106 @@
...
-    "rent-ride-fe": {
-      "projectType": "application",
-      "schematics": {
-        "@schematics/angular:component": {
-          "style": "scss"
-        }
-      },
-      "root": "",
-      "sourceRoot": "src",
-      "prefix": "app",
```
Comment (phucdang01235) [inline angular.json:15]: dfgdfgd

**Review by phucdang01235 — CHANGES_REQUESTED** [review]
Reuqets

File: angular.json (hunk 3 above)
Comment (dangthanhphuc) [reply angular.json:15]: sdfswef

**Review by phucdang01235 — APPROVED** [review]

**Pull request merged**